### DISCUSSION TITLE
NextJs + tRPC integration of Highlight

### DISCUSSION THREAD

**Discussion by seedy** [highlight/highlight#6403] [Q&A] [answered]
Hi there 👋 

I'm just trying out highlight as a drop-in replacement to sentry. I'm thrilled !

I am looking for the right integration of highlight for NextJs with tRPC integration of API routes.

[From the docs](https://www.highlight.io/docs/getting-started/backend-sdk/js/trpc), it seems I need both `@highlight-run/next` and `@highlight-run/node` to make my environment work.

I feel like it might be suboptimal to require both packages for my next project ?

Would it be possible to have the trpc error handler part of next package ?

Thanks in advance for the feedback 😁

**Comment by jay-khatri** [comment, marked as answer]
Hi there! In the short term, we'll likely stick with the current set up because it makes our build process a little easier to maintain. But we'll note this down when we make a change to our Sdk architecture.
